_template/bin/nuget

```diff
@@ -1,2 +1,6 @@
-result = system(File.dirname(__FILE__) + "/nuget.exe " + ARGV.join(' '))
-exit 1 unless result
\ No newline at end of file
+host_os = RbConfig::CONFIG['host_os']
+windows = /mswin|msys|mingw|cygwin|bccwin|wince|emc/.match(host_os)
```

Comment:
It looks like env os works for modern ruby installations:
http://stackoverflow.com/questions/4871309/what-is-the-correct-way-to-detect-if-ruby-is-running-on-windows